index.html

```diff
@@ -0,0 +1,359 @@
+    <link rel="stylesheet" href="js/animation.js" />
+    <link rel="stylesheet" href="css/bootstrap-grid.min.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Aleo&family=Josefin+Sans:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="menu_burger">
+        <div class="menu-btn">
+          <div class="line line__1"></div>
+          <div class="line line__2"></div>
+          <div class="line line__3"></div>
+        </div>
+
+        <div class="sub-menu-btn">
+          <div class="line line__1"></div>
+          <div class="line line__2"></div>
+        </div>
+        <img class="grogu" src="pngaaa.png" alt="TheChild" />
+
+        <ul class="nav-links">
+          <li class="link">
+            <a class="responsive_link" href="#">Who We Are</a>
+          </li>
+          <li class="link">
+            <a class="responsive_link" href="#">What We Do</a>
+          </li>
+          <li class="link">
+            <a class="responsive_link" href="#">Travel With Us</a>
+          </li>
+          <li class="link">
+            <a class="responsive_link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <h1>Bhromaon</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12">
+            <p>Safe Travel Anywhere</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12">
+            <div class="search">
+              <input
+                type="search"
+                placeholder="Search Your Travel Destination"
+                name="q"
+              />
+              <button name="submit" class="header_button" type="submit">
+                <span class="icon-search"></span> Search
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12">
+            <p class="scroll">Scroll Down</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12"><span class="icon-chevron-down"></span></div>
+        </div>
+      </div>
+    </header>
+    <section class="nav">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-2 col-md-12">
+            <p>Bhromaon</p>
+          </div>
+          <div class="col-lg-10 col-md-12">
+            <ul class="menu-main">
+              <li><a href="#" class="first_link">Who We Are</a></li>
+              <li><a href="#">What We Do</a></li>
+              <li><a href="#">Travel With Us</a></li>
+              <li><a href="#">Contact Us</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="WhyChooseUs">
```

Comment:
лучше придерживаться одного стиля в названиях классов. Например часто используют **kebab-case**
```suggestion
    <section class="why-choose-us">
```